samza-sql/src/main/java/org/apache/samza/sql/impl/ConfigBasedIOResolverFactory.java

```diff
@@ -68,10 +68,10 @@ public ConfigBasedIOResolver(Config config) {
 
     @Override
     public SqlIOConfig fetchSourceInfo(String source) {
-      return fetchSystemInfo(source);
+      return fetchSystemInfo(source, false);
     }
 
-    private SqlIOConfig fetchSystemInfo(String name) {
+    private SqlIOConfig fetchSystemInfo(String name, boolean isSink) {
```

Comment:
Are we using the sink or source info?  If not, Can we remove this and add it when we are going to use it?

Reply:
Sure.